cryocloud link and logo fix.

markweldensmith opened the pull request.

markweldensmith committed:
cryocloud link and logo fix.

aaarendt reviewed (changes requested):
On reviewing this I realized another change is needed: we have hard-coded links to CryoCloud in https://github.com/ICESAT-2HackWeek/website-2025/blob/main/book/preliminary/checklist.md. 

The URL on line 62 of `checklist.md` should be `https://book.cryointhecloud.com`
The URL on line 63 should be `https://book.cryointhecloud.com/getting-started`

markweldensmith committed:
Fix cryocloud links for checklist.md

Updated broken links to cryocloud resources.

markweldensmith committed:
Updated cryocloud link following review of cookiecutter.yaml

Changed the URL from https://book.cryointhecloud.com/getting-started to https://book.cryointhecloud.com

markweldensmith commented on cookiecutter.yaml:
```diff
@@ -45,12 +45,12 @@ sponsors:
     website: 'https://icesat-2.gsfc.nasa.gov/mission'
     logo_url: https://github.com/ICESAT-2HackWeek/icesat-2hackweek.github.io/raw/2022.03.01/assets/images/ICESat2.png
+  - name: CryoCloud
+    website: https://book.cryointhecloud.com/getting-started
```
Changes made to cryocloud link to provide root level access: https://book.cryointhecloud.com

aaarendt reviewed (approved):
Looks great!
